megamek/src/megamek/common/Player.java

```diff
@@ -467,6 +467,75 @@ public int getEntityCount() {
                 .filter(entity -> !entity.isDestroyed() && !entity.isTrapped()).count());
     }
 
+    public int getUnitCount() {
+        return Math.toIntExact(game.getPlayerEntities(this, false).stream()
+                .filter(entity -> !entity.isDestroyed() && !entity.isTrapped() && !(entity instanceof EjectedCrew)).count());
+    }
+
+    public int getUnitLightDamageCount() {
```

Comment:
It seems like using the damage type as a parameter here would save some repetition.
int getDamagedUnits(int damage) ...... entity.getDamageLevel() == damage